lib/web/apiserver.go

```diff
@@ -860,14 +850,10 @@ func (h *Handler) oidcLoginWeb(w http.ResponseWriter, r *http.Request, p httprou
 			CheckUser:         true,
 		})
 	if err != nil {
-		// redirect to an error page
-		pathToError := url.URL{
-			Path:     "/web/msg/error/login_failed",
-			RawQuery: url.Values{"details": []string{err.Error()}}.Encode(),
-		}
-		http.Redirect(w, r, pathToError.String(), http.StatusFound)
-		return nil, nil
+		logger.WithError(err).Error("Error with creating auth request.")
```

Comment:
To match other errors:

```suggestion
		logger.WithError(err).Error("Error creating auth request.")
```

Reply:
revised